crates/static-analysis-kernel/src/analysis/ddsa_lib/context/file.rs

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License, Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+#[derive(Debug, Default)]
+pub struct FileContext {
+    // Supported file contexts:
+}
+
+impl FileContext {
+    //
+}
```

Comment:
We already have the `FileContext` for language-specific feature. Are you reusing it, killing it? What do we want to do here?